sql/core/src/test/scala/org/apache/spark/sql/execution/benchmark/TPCDSQueryBenchmark.scala

```diff
@@ -99,6 +95,20 @@ object TPCDSQueryBenchmark {
   }
 
   def main(args: Array[String]): Unit = {
+    if (args.length < 1) {
+      // scalastyle:off
+      println(
+        s"""
+           |Usage: spark-submit --class <this class> <spark sql test jar> <TPCDS data location>
+           |
+           |In order to run this benchmark, please follow the instructions at
+           |https://github.com/databricks/spark-sql-perf/blob/master/README.md
```

Comment:
[I a bit played around the generator part](https://github.com/maropu/spark-tpcds-datagen) in `spark-sql-perf` and I think the part is small among the package. So, any plan to make a new repository to generate the data only in the databricks repository or something?